build.gradle

```diff
@@ -5,13 +5,20 @@ buildscript {
 
     dependencies {
         classpath 'com.github.jengelman.gradle.plugins:shadow:2.0.1'
+        classpath 'org.ajoberstar:gradle-git-publish:0.3.2'
+        classpath("org.ajoberstar:grgit:2.1.0")
```

Comment:
Remove the brackets so it will look consistent with other deps.

Reply:
3e8593d9